crypto/s2n_ecc.c

```diff
@@ -224,15 +242,13 @@ static int s2n_ecc_write_point_data_snug(const EC_POINT * point, const EC_GROUP
     return 0;
 }
 
-static int s2n_ecc_write_point_with_length(const EC_POINT * point, const EC_GROUP * group, struct s2n_stuffer *out)
+static int s2n_ecc_write_point(const EC_POINT * point, const EC_GROUP * group, struct s2n_stuffer *out)
 {
     uint8_t point_len;
     struct s2n_blob point_blob = {0};
 
     GUARD(s2n_ecc_calculate_point_length(point, group, &point_len));
 
-    GUARD(s2n_stuffer_write_uint8(out, point_len));
```

Comment:
have this line been moved a different location?

Reply:
Yes. Previously everything that needed to write a point wrote a 1-byte length in front of the point parameters, but the new extension writes a 2-byte length. I therefore moved the length-writing up to the individual callers of this method.